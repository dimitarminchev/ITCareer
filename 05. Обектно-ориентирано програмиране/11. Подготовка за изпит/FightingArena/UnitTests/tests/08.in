CreateHero:Assassin:Filip
CreateWeapon:Pesho:Sword:Name:5:5:5
CreateHero:Warrior:Simo
Fight:Simo:Dimo
HeroInfo:Filip
Fight:Filip:Simo
CloseArena