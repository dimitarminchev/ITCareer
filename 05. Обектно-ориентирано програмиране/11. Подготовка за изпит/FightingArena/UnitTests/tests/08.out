Assassin: Filip joined the Arena!
Hero with name: Pesho does not exist!
Warrior: Simo joined the Arena!
Hero with name: Dimo does not exist!
Assassin: Filip - Level: 0
Experience: 0
Power: 0.00
No winner in battle between Filip and Simo!
Heroes: 2
Assassin: Filip - Level: 0
Experience: 15
Power: 0.00
Warrior: Simo - Level: 0
Experience: 15
Power: 0.00